test/plugin/test_out_google_cloud.rb

```diff
@@ -127,6 +146,18 @@ def setup
     }
   }
 
+  EC2_PARAMS = {
+    'service_name' => EC2_SERVICE_NAME,
+    'log_name' => 'test',
+    'project_id' => EC2_PROJECT_ID,
+    'zone' => EC2_PREFIXED_ZONE,
+    'labels' => {
+      "#{EC2_SERVICE_NAME}/resource_type" => 'instance',
+      "#{EC2_SERVICE_NAME}/resource_id" => EC2_VM_ID,
+      "#{EC2_SERVICE_NAME}/account_id" => EC2_ACCOUNT_ID
```

Comment:
Is it important to test that this will work when "account_id" in not in metadata returned by EC2?